packages/functions/src/dataTriggers.ts

```diff
@@ -269,7 +269,82 @@ export const createAttendanceForBooking = functions
 
     await attendanceRef.set(updatedEntry, { merge: true });
   });
+// SlotAttendnace = {
+//   /**
+//    * ISO date of the slot for easier querying
+//    */
+//   date: string;
+//   /**
+//    * Attendances for slot keyed by customer
+//    */
+//   attendances: {
+//     [customerId: string]: CustomerAttendance;
+//   };
+// };
+
+// CustomerAttendance {
+//   bookedInterval: string | null;
+//   attendedInterval: string | null;
+// }
+
+// entry
+// [slotId: string]: CustomerBookingEntry;
+// {
+//   /**
+//    * ISO date of the booked slot (used for easier querying from store)
+//    */
+//   date: string;
+//   /**
+//    * Booked interval for particular slot
+//    */
+//   interval: string;
+
+// }
+export const createAttendedSlotOnAttendance = functions
+  .region(__functionsZone__)
+  .firestore.document(
+    `${Collection.Organizations}/{organization}/${OrgSubCollection.Attendance}/{bookingId}/`
```

Comment:
Let's call the document id `slotId`, as it's used to key the attendance entries, but also to group (and point back to) the given slot.